WARNING 2018-12-05 11:04:57,013 base 93 Not Found: /sms_code/15766205544
WARNING 2018-12-05 11:04:57,016 basehttp 124 "GET /sms_code/15766205544 HTTP/1.1" 404 2450
WARNING 2018-12-05 11:04:57,125 base 93 Not Found: /favicon.ico
WARNING 2018-12-05 11:04:57,126 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2423
WARNING 2018-12-05 11:05:07,053 base 93 Not Found: /sms_code/157662055441
WARNING 2018-12-05 11:05:07,054 basehttp 124 "GET /sms_code/157662055441 HTTP/1.1" 404 2453
WARNING 2018-12-05 11:05:13,640 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:05:13,641 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2453
WARNING 2018-12-05 11:06:57,383 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:06:57,384 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2594
WARNING 2018-12-05 11:07:30,613 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:07:30,615 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2594
WARNING 2018-12-05 11:11:32,871 base 93 Not Found: /sms_code/
WARNING 2018-12-05 11:11:32,873 basehttp 124 "GET /sms_code/?mobile=15766205544/ HTTP/1.1" 404 2578
WARNING 2018-12-05 11:12:16,409 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:12:16,410 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2465
WARNING 2018-12-05 11:12:21,510 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:12:21,511 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2465
WARNING 2018-12-05 11:12:38,696 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:12:38,697 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2465
WARNING 2018-12-05 11:12:56,249 base 93 Not Found: /sms_code/
WARNING 2018-12-05 11:12:56,250 basehttp 124 "GET /sms_code/?mobile=15766205544/ HTTP/1.1" 404 2449
WARNING 2018-12-05 11:13:06,051 base 93 Not Found: /sms_code/
WARNING 2018-12-05 11:13:06,052 basehttp 124 "GET /sms_code/?mobile=15766205544/ HTTP/1.1" 404 2449
WARNING 2018-12-05 11:14:43,484 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:14:43,486 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2453
WARNING 2018-12-05 11:19:02,807 base 93 Not Found: /sms_code/15766205544/
WARNING 2018-12-05 11:19:02,808 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 404 2452
INFO 2018-12-05 11:19:35,816 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5244
INFO 2018-12-05 11:19:36,008 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-05 11:19:36,061 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-05 11:19:36,061 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-05 11:19:36,074 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-05 11:19:36,079 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-05 11:19:36,082 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-05 11:19:36,084 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-05 11:19:36,126 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-05 11:19:36,126 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-05 11:19:36,127 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-05 11:19:36,188 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2018-12-05 11:21:21,147 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5244
INFO 2018-12-05 11:22:51,994 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5244
INFO 2018-12-05 11:23:40,359 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5258
INFO 2018-12-05 11:23:46,349 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5258
INFO 2018-12-05 11:24:26,863 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5244
INFO 2018-12-05 11:26:08,423 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5270
INFO 2018-12-05 11:28:02,798 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5270
INFO 2018-12-05 11:28:05,479 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5258
INFO 2018-12-05 11:33:18,693 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5270
INFO 2018-12-05 11:33:35,511 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5258
INFO 2018-12-05 11:34:02,608 basehttp 124 "GET /sms_code/1576620544/ HTTP/1.1" 200 5275
INFO 2018-12-05 11:34:29,816 basehttp 124 "GET /sms_code/15766205644/ HTTP/1.1" 200 5270
INFO 2018-12-05 17:02:32,113 basehttp 124 "GET /sms_code/15766205644/ HTTP/1.1" 200 5270
INFO 2018-12-05 17:02:32,135 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,137 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,139 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,144 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,145 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,149 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,150 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,154 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,156 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,157 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2018-12-05 17:02:32,349 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2018-12-05 17:02:41,380 basehttp 124 "GET /sms_code/15766205644/ HTTP/1.1" 200 5258
ERROR 2018-12-05 17:16:12,434 exception 118 Internal Server Error: /username/admin/count/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 156, in _get_response
    response = response.render()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\template\response.py", line 106, in render
    self.content = self.rendered_content
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 681, in get_context
    'content': self.get_content(renderer, data, accepted_media_type, renderer_context),
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 422, in get_content
    content = renderer.render(data, accepted_media_type, renderer_context)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 105, in render
    allow_nan=not self.strict, separators=separators
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\utils\json.py", line 28, in dumps
    return json.dumps(*args, **kwargs)
  File "F:\Python\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "F:\Python\lib\json\encoder.py", line 201, in encode
    chunks = list(chunks)
  File "F:\Python\lib\json\encoder.py", line 430, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "F:\Python\lib\json\encoder.py", line 404, in _iterencode_dict
    yield from chunks
  File "F:\Python\lib\json\encoder.py", line 438, in _iterencode
    yield from _iterencode(o, _current_indent_level)
  File "F:\Python\lib\json\encoder.py", line 428, in _iterencode
    yield from _iterencode_list(o, _current_indent_level)
  File "F:\Python\lib\json\encoder.py", line 325, in _iterencode_list
    yield from chunks
  File "F:\Python\lib\json\encoder.py", line 437, in _iterencode
    o = _default(o)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\utils\encoders.py", line 68, in default
    return super(JSONEncoder, self).default(obj)
  File "F:\Python\lib\json\encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'User' is not JSON serializable
ERROR 2018-12-05 17:16:12,980 basehttp 124 "GET /username/admin/count/ HTTP/1.1" 500 166913
INFO 2018-12-05 17:17:14,825 basehttp 124 "GET /username/admin/count/ HTTP/1.1" 200 5236
INFO 2018-12-05 17:27:33,348 basehttp 124 "GET /mobile/15766205544/count/ HTTP/1.1" 200 5243
INFO 2018-12-05 17:27:39,294 basehttp 124 "GET /mobile/1576620554/count/ HTTP/1.1" 200 5236
INFO 2018-12-05 17:27:43,094 basehttp 124 "GET /mobile/15766205544/count/ HTTP/1.1" 200 5243
WARNING 2018-12-05 17:50:37,468 base 93 Not Found: /
WARNING 2018-12-05 17:50:37,470 basehttp 124 "GET / HTTP/1.1" 404 2780
INFO 2018-12-05 17:53:37,344 basehttp 124 "GET / HTTP/1.1" 200 5347
ERROR 2018-12-05 17:53:40,154 exception 118 Internal Server Error: /users/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "F:\Python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'meiduo.utils'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ModuleNotFoundError: No module named 'meiduo.utils'.
ERROR 2018-12-05 17:53:40,760 basehttp 124 "GET /users/ HTTP/1.1" 500 130980
ERROR 2018-12-05 17:57:30,787 exception 118 Internal Server Error: /users/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "F:\Python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'meiduo.utils'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ModuleNotFoundError: No module named 'meiduo.utils'.
ERROR 2018-12-05 17:57:30,841 basehttp 124 "GET /users/ HTTP/1.1" 500 130973
ERROR 2018-12-05 17:59:09,584 exception 118 Internal Server Error: /users/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 156, in _get_response
    response = response.render()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\template\response.py", line 106, in render
    self.content = self.rendered_content
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 656, in get_context
    raw_data_post_form = self.get_raw_data_form(data, view, 'POST', request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\renderers.py", line 572, in get_raw_data_form
    data = serializer.data.copy()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 560, in data
    ret = super(Serializer, self).data
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 266, in data
    self._data = self.get_initial()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 413, in get_initial
    for field in self.fields.values()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 1054, in get_fields
    fields[field_name] = field_class(**field_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\fields.py", line 754, in __init__
    super(CharField, self).__init__(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'write_obly'
ERROR 2018-12-05 17:59:10,041 basehttp 124 "GET /users/ HTTP/1.1" 500 125358
WARNING 2018-12-05 18:01:23,166 basehttp 124 "GET /users/ HTTP/1.1" 405 8997
WARNING 2018-12-05 19:30:37,890 basehttp 124 "POST /users/ HTTP/1.1" 400 9331
WARNING 2018-12-05 19:34:49,376 basehttp 124 "POST /users/ HTTP/1.1" 400 9163
INFO 2018-12-05 19:37:31,890 basehttp 124 "GET / HTTP/1.1" 200 5347
WARNING 2018-12-05 19:37:34,097 basehttp 124 "GET /users/ HTTP/1.1" 405 8727
INFO 2018-12-06 11:00:34,805 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:00:34,926 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
WARNING 2018-12-06 11:00:35,173 base 93 Not Found: /favicon.ico
WARNING 2018-12-06 11:00:35,175 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4378
WARNING 2018-12-06 11:00:37,785 basehttp 124 "GET /users/ HTTP/1.1" 405 8727
INFO 2018-12-06 11:03:31,991 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5363
INFO 2018-12-06 11:04:06,956 basehttp 124 "POST /users/ HTTP/1.1" 201 8982
INFO 2018-12-06 11:06:16,417 basehttp 124 "GET / HTTP/1.1" 200 5347
WARNING 2018-12-06 11:06:17,650 basehttp 124 "GET /users/ HTTP/1.1" 405 8727
INFO 2018-12-06 11:08:10,243 basehttp 124 "GET / HTTP/1.1" 200 5347
WARNING 2018-12-06 11:08:12,245 basehttp 124 "GET /users/ HTTP/1.1" 405 8727
INFO 2018-12-06 11:12:42,241 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:12:43,637 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:12:43,757 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-06 11:12:43,760 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-06 11:12:43,763 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-06 11:12:43,763 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-06 11:12:43,767 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-06 11:12:43,769 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-06 11:12:43,770 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-06 11:12:43,771 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-06 11:12:43,772 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-06 11:12:43,775 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-06 11:12:43,825 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-06 11:12:43,853 base 93 Not Found: /favicon.ico
WARNING 2018-12-06 11:12:43,854 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4378
WARNING 2018-12-06 11:12:45,474 basehttp 124 "GET /users/ HTTP/1.1" 405 8731
INFO 2018-12-06 11:13:19,329 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5363
INFO 2018-12-06 11:13:45,003 basehttp 124 "POST /users/ HTTP/1.1" 201 8986
INFO 2018-12-06 11:17:02,669 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:17:04,325 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:17:04,357 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-06 11:17:04,358 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-06 11:17:04,359 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-06 11:17:04,364 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-06 11:17:04,366 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-06 11:17:04,368 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-06 11:17:04,373 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-06 11:17:04,373 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-06 11:17:04,374 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-06 11:17:04,375 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-06 11:17:04,395 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-06 11:17:04,450 base 93 Not Found: /favicon.ico
WARNING 2018-12-06 11:17:04,451 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4378
INFO 2018-12-06 11:17:06,072 basehttp 124 "GET / HTTP/1.1" 200 5347
INFO 2018-12-06 11:17:06,109 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-06 11:17:06,116 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-06 11:17:06,123 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-06 11:17:06,124 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-06 11:17:06,125 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-06 11:17:06,129 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-06 11:17:06,132 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-06 11:17:06,133 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-06 11:17:06,134 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-06 11:17:06,135 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-06 11:17:06,157 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-06 11:17:06,207 base 93 Not Found: /favicon.ico
WARNING 2018-12-06 11:17:06,208 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4378
INFO 2018-12-06 11:17:53,515 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5363
WARNING 2018-12-06 11:17:58,234 basehttp 124 "GET /users/ HTTP/1.1" 405 8731
INFO 2018-12-06 11:18:35,451 basehttp 124 "POST /users/ HTTP/1.1" 201 8986
INFO 2018-12-06 11:37:54,288 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5363
INFO 2018-12-06 11:38:23,231 basehttp 124 "POST /users/ HTTP/1.1" 201 8986
INFO 2018-12-06 11:40:18,716 basehttp 124 "GET /sms_code/15766205544/ HTTP/1.1" 200 5363
WARNING 2018-12-06 11:40:31,108 basehttp 124 "POST /users/ HTTP/1.1" 400 8895
INFO 2018-12-06 11:40:53,397 basehttp 124 "POST /users/ HTTP/1.1" 201 8986
INFO 2018-12-06 13:59:34,204 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 204
INFO 2018-12-07 10:02:07,996 basehttp 124 "GET / HTTP/1.1" 200 5344
INFO 2018-12-07 10:02:08,138 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,162 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,177 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,178 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,179 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,220 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,221 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,222 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,222 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,223 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2018-12-07 10:02:08,302 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
WARNING 2018-12-07 10:02:08,469 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 10:02:08,470 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 5174
INFO 2018-12-07 10:02:12,764 basehttp 124 "GET /email HTTP/1.1" 301 0
WARNING 2018-12-07 10:02:12,941 basehttp 124 "GET /email/ HTTP/1.1" 405 7338
ERROR 2018-12-07 10:04:58,681 exception 118 Internal Server Error: /email/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\generics.py", line 228, in put
    return self.update(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 70, in update
    self.perform_update(serializer)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 80, in perform_update
    serializer.save()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 209, in save
    self.instance = self.update(self.instance, validated_data)
  File "F:\Pycharm\Project\meiduo_env\meiduo\apps\users\serializers.py", line 112, in update
    instance.save()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\models.py", line 387, in save
    raise NotImplementedError("Django doesn't provide a DB representation for AnonymousUser.")
NotImplementedError: Django doesn't provide a DB representation for AnonymousUser.
ERROR 2018-12-07 10:04:59,312 basehttp 124 "PUT /email/ HTTP/1.1" 500 116835
INFO 2018-12-07 10:05:31,352 basehttp 124 "GET /api-auth/login/?next=/ HTTP/1.1" 200 2702
INFO 2018-12-07 10:05:40,813 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 302 0
INFO 2018-12-07 10:05:40,873 basehttp 124 "GET / HTTP/1.1" 200 5572
WARNING 2018-12-07 10:06:34,220 basehttp 124 "GET /email/ HTTP/1.1" 405 7566
INFO 2018-12-07 10:07:02,551 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
WARNING 2018-12-07 10:21:40,831 basehttp 124 "GET /email/ HTTP/1.1" 405 7566
INFO 2018-12-07 10:21:47,029 basehttp 124 "PUT /email/ HTTP/1.1" 200 7517
INFO 2018-12-07 10:22:25,847 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:38,385 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:40,484 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:40,730 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:41,092 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:45,123 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:22:51,820 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:25:59,819 basehttp 124 "GET / HTTP/1.1" 200 5572
WARNING 2018-12-07 10:26:04,741 basehttp 124 "GET /email/ HTTP/1.1" 405 7566
INFO 2018-12-07 10:26:33,628 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:37:04,146 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 10:37:05,637 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 10:37:05,761 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-07 10:37:05,764 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-07 10:37:05,766 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-07 10:37:05,775 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-07 10:37:05,779 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-07 10:37:05,780 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-07 10:37:05,786 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-07 10:37:05,790 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-07 10:37:05,803 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-07 10:37:05,808 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-07 10:37:05,826 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-07 10:37:05,887 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 10:37:05,888 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 5341
WARNING 2018-12-07 10:37:09,988 basehttp 124 "GET /email/ HTTP/1.1" 405 7566
INFO 2018-12-07 10:37:36,363 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:38:52,639 basehttp 124 "GET /emails/verification/ HTTP/1.1" 200 5617
ERROR 2018-12-07 10:40:02,493 exception 118 Internal Server Error: /emails/verification/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 485, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 400, in finalize_response
    % type(response)
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
ERROR 2018-12-07 10:40:02,789 basehttp 124 "GET /emails/verification/?token=eyJhbGciOiJIUzI1NiIsImlhdCI6MTU0NDE1MDIzMywiZXhwIjoxNTQ0MTUwNTMzfQ.eyJ1c2VybmFtZSI6ImFkbWluIn0.KzK25JrQV-YdmStVBaJXepIjH8aKqxsLxsPwOnQ1hsI HTTP/1.1" 500 87785
ERROR 2018-12-07 10:44:07,290 exception 74 Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS.
WARNING 2018-12-07 10:44:07,662 basehttp 124 "GET / HTTP/1.1" 400 69934
ERROR 2018-12-07 10:44:07,956 exception 74 Invalid HTTP_HOST header: 'localhost:8000'. You may need to add 'localhost' to ALLOWED_HOSTS.
WARNING 2018-12-07 10:44:07,993 basehttp 124 "GET /favicon.ico HTTP/1.1" 400 70012
INFO 2018-12-07 10:45:21,188 basehttp 124 "GET / HTTP/1.1" 200 5344
INFO 2018-12-07 10:45:21,208 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-07 10:45:21,388 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-07 10:45:21,514 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-07 10:45:21,518 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-07 10:45:21,518 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-07 10:45:21,519 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-07 10:45:21,520 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-07 10:45:21,695 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-07 10:45:21,820 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-07 10:45:21,844 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-07 10:45:21,932 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-07 10:45:22,151 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 10:45:22,152 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 5341
INFO 2018-12-07 10:45:27,135 basehttp 124 "GET /email HTTP/1.1" 301 0
WARNING 2018-12-07 10:45:27,497 basehttp 124 "GET /email/ HTTP/1.1" 405 7338
ERROR 2018-12-07 10:45:31,134 exception 118 Internal Server Error: /email/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\generics.py", line 228, in put
    return self.update(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 70, in update
    self.perform_update(serializer)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 80, in perform_update
    serializer.save()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\serializers.py", line 209, in save
    self.instance = self.update(self.instance, validated_data)
  File "F:\Pycharm\Project\meiduo_env\meiduo\apps\users\serializers.py", line 112, in update
    instance.save()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\models.py", line 387, in save
    raise NotImplementedError("Django doesn't provide a DB representation for AnonymousUser.")
NotImplementedError: Django doesn't provide a DB representation for AnonymousUser.
ERROR 2018-12-07 10:45:31,440 basehttp 124 "PUT /email/ HTTP/1.1" 500 116845
INFO 2018-12-07 10:45:40,042 basehttp 124 "GET /api-auth/login/?next=/ HTTP/1.1" 200 2702
INFO 2018-12-07 10:45:41,478 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 302 0
INFO 2018-12-07 10:45:41,693 basehttp 124 "GET / HTTP/1.1" 200 5572
WARNING 2018-12-07 10:45:46,269 base 93 Not Found: /e
WARNING 2018-12-07 10:45:46,271 basehttp 124 "GET /e HTTP/1.1" 404 5311
WARNING 2018-12-07 10:45:52,016 basehttp 124 "GET /email/ HTTP/1.1" 405 7566
INFO 2018-12-07 10:46:14,232 basehttp 124 "PUT /email/ HTTP/1.1" 200 7603
INFO 2018-12-07 10:46:43,835 basehttp 124 "GET /emails/verification/?token=eyJhbGciOiJIUzI1NiIsImlhdCI6MTU0NDE1MDc1NSwiZXhwIjoxNTQ0MTUxMDU1fQ.eyJ1c2VybmFtZSI6ImFkbWluIn0.o8hCsnnQkXMqhNVQ8PlPbLeka3Fc2gceHWgLcK2OJ3Q HTTP/1.1" 200 6333
INFO 2018-12-07 11:30:29,932 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:30:36,074 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:32:37,922 basehttp 124 "GET /areas/110100 HTTP/1.1" 301 0
ERROR 2018-12-07 11:32:37,972 exception 118 Internal Server Error: /areas/110100/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
TypeError: inner() missing 1 required positional argument: 'request'
ERROR 2018-12-07 11:32:38,298 basehttp 124 "GET /areas/110100/ HTTP/1.1" 500 98322
INFO 2018-12-07 11:33:49,704 basehttp 124 "GET /areas/110100/ HTTP/1.1" 200 7061
INFO 2018-12-07 11:36:21,799 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:36:23,200 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:36:45,476 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:37:49,527 basehttp 124 "GET /areas HTTP/1.1" 301 0
INFO 2018-12-07 11:37:49,585 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:38:19,114 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:39:17,617 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:39:25,737 basehttp 124 "GET /areas/110100/ HTTP/1.1" 200 7061
INFO 2018-12-07 11:39:39,679 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:40:08,945 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:40:10,635 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:40:10,662 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-07 11:40:10,663 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-07 11:40:10,664 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-07 11:40:10,666 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-07 11:40:10,677 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-07 11:40:10,683 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-07 11:40:10,685 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-07 11:40:10,687 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-07 11:40:10,687 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-07 11:40:10,688 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-07 11:40:10,710 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
WARNING 2018-12-07 11:40:10,774 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 11:40:10,775 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 6315
INFO 2018-12-07 11:40:12,373 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:40:21,335 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:40:26,946 basehttp 124 "GET /areas/110100/ HTTP/1.1" 200 7061
WARNING 2018-12-07 11:40:27,135 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 11:40:27,137 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 6315
INFO 2018-12-07 11:40:34,762 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:40:39,201 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:40:40,714 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:41:20,870 basehttp 124 "GET /areas/?parent_id=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:41:24,318 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:43:10,341 basehttp 124 "GET /areas/?parentid=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:44:18,241 basehttp 124 "GET /areas/?parentid=110100 HTTP/1.1" 200 8783
INFO 2018-12-07 11:44:56,968 basehttp 124 "GET / HTTP/1.1" 200 5572
INFO 2018-12-07 11:44:58,425 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:45:26,460 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:45:26,494 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-07 11:45:26,694 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-07 11:45:26,796 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-07 11:45:26,797 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-07 11:45:26,820 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-07 11:45:26,821 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-07 11:45:26,826 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-07 11:45:27,003 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-07 11:45:27,113 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-07 11:45:27,118 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2018-12-07 11:45:27,153 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
WARNING 2018-12-07 11:45:27,311 base 93 Not Found: /favicon.ico
WARNING 2018-12-07 11:45:27,313 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 6315
INFO 2018-12-07 11:45:32,375 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:45:41,113 basehttp 124 "GET / HTTP/1.1" 200 5344
INFO 2018-12-07 11:45:41,358 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-07 11:45:41,439 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-07 11:45:41,442 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-07 11:45:41,443 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-07 11:45:41,445 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-07 11:45:41,445 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-07 11:45:41,663 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-07 11:45:41,752 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-07 11:45:41,753 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-07 11:45:41,756 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2018-12-07 11:45:41,758 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-07 11:45:51,638 basehttp 124 "GET /areas/ HTTP/1.1" 200 8783
INFO 2018-12-07 11:47:54,695 basehttp 124 "GET / HTTP/1.1" 200 5476
INFO 2018-12-07 11:47:56,301 basehttp 124 "GET /areaview/ HTTP/1.1" 200 8480
INFO 2018-12-07 11:48:33,757 basehttp 124 "GET /areaview/parantid=110100 HTTP/1.1" 301 0
ERROR 2018-12-07 11:48:34,019 exception 118 Internal Server Error: /areaview/parantid=110100/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework_extensions\cache\decorators.py", line 48, in inner
    kwargs=kwargs,
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework_extensions\cache\decorators.py", line 67, in process_cache_response
    response = view_method(view_instance, request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework_extensions\cache\mixins.py", line 22, in retrieve
    return super(RetrieveCacheResponseMixin, self).retrieve(request, *args, **kwargs)
AttributeError: 'super' object has no attribute 'retrieve'
ERROR 2018-12-07 11:48:34,388 basehttp 124 "GET /areaview/parantid%3D110100/ HTTP/1.1" 500 104651
INFO 2018-12-07 11:49:42,171 basehttp 124 "GET /areaview/ HTTP/1.1" 200 8480
INFO 2018-12-07 11:49:52,777 basehttp 124 "GET /areaview/?parent_id=110100 HTTP/1.1" 200 8480
INFO 2018-12-07 11:50:39,584 basehttp 124 "GET /areaview/?parent_id=110100 HTTP/1.1" 200 8480
INFO 2018-12-07 11:51:02,335 basehttp 124 "GET /areaview/?parent_id=110100 HTTP/1.1" 200 8480
INFO 2018-12-07 11:51:16,328 basehttp 124 "GET /areaview/?parent_id=110100 HTTP/1.1" 200 6824
INFO 2018-12-07 11:51:50,455 basehttp 124 "GET /areas/110100 HTTP/1.1" 301 0
INFO 2018-12-07 11:51:50,743 basehttp 124 "GET /areas/110100/ HTTP/1.1" 200 6833
INFO 2018-12-07 14:22:26,691 basehttp 124 "GET / HTTP/1.1" 200 5482
ERROR 2018-12-07 14:22:29,837 exception 118 Internal Server Error: /address/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\meiduo\apps\areas\views.py", line 55, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\meiduo\apps\areas\views.py", line 52, in get_queryset
    return Address.objects.filter(user=self.request.user, is_deleted=False)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\query.py", line 836, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\query.py", line 854, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\sql\query.py", line 1253, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\sql\query.py", line 1277, in _add_q
    split_subq=split_subq,
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\sql\query.py", line 1215, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\sql\query.py", line 1085, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\lookups.py", line 18, in __init__
    self.rhs = self.get_prep_lookup()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\fields\related_lookups.py", line 115, in get_prep_lookup
    self.rhs = target_field.get_prep_value(self.rhs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\fields\__init__.py", line 947, in get_prep_value
    return int(value)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'AnonymousUser'
ERROR 2018-12-07 14:22:30,304 basehttp 124 "GET /address/ HTTP/1.1" 500 146631
INFO 2018-12-07 14:24:31,450 basehttp 124 "GET / HTTP/1.1" 200 5482
ERROR 2018-12-07 14:24:33,620 exception 118 Internal Server Error: /address/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 471, in dispatch
    self.initial(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 389, in initial
    self.check_permissions(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 321, in check_permissions
    for permission in self.get_permissions():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 268, in get_permissions
    return [permission() for permission in self.permission_classes]
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 268, in <listcomp>
    return [permission() for permission in self.permission_classes]
TypeError: 'IsAuthenticated' object is not callable
ERROR 2018-12-07 14:24:33,939 basehttp 124 "GET /address/ HTTP/1.1" 500 105712
WARNING 2018-12-07 14:25:42,964 basehttp 124 "GET /address/ HTTP/1.1" 401 5152
INFO 2018-12-07 14:25:46,307 basehttp 124 "GET /api-auth/login/?next=/address/ HTTP/1.1" 200 2710
INFO 2018-12-07 14:25:51,704 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 302 0
INFO 2018-12-07 14:25:51,964 basehttp 124 "GET /address/ HTTP/1.1" 200 11282
INFO 2018-12-07 14:39:40,391 basehttp 124 "GET /address/ HTTP/1.1" 200 97233
INFO 2018-12-07 14:41:07,803 basehttp 124 "GET /address/ HTTP/1.1" 200 97233
INFO 2018-12-07 14:42:44,987 basehttp 124 "GET /address/ HTTP/1.1" 200 97886
INFO 2018-12-07 14:44:43,311 basehttp 124 "GET /address/ HTTP/1.1" 200 11282
INFO 2018-12-07 15:09:34,600 basehttp 124 "GET / HTTP/1.1" 200 5710
INFO 2018-12-07 15:09:41,563 basehttp 124 "GET /areadump HTTP/1.1" 200 236544
INFO 2018-12-07 15:16:42,295 basehttp 124 "GET / HTTP/1.1" 200 5710
INFO 2018-12-07 15:16:53,978 basehttp 124 "GET /areadump HTTP/1.1" 200 240640
INFO 2018-12-07 15:23:38,567 basehttp 124 "GET /areadump HTTP/1.1" 200 240640
INFO 2018-12-10 10:00:27,098 basehttp 124 "GET / HTTP/1.1" 200 5710
INFO 2018-12-10 10:00:27,116 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2018-12-10 10:00:27,116 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
WARNING 2018-12-10 10:00:29,122 base 93 Not Found: /admin
WARNING 2018-12-10 10:00:29,125 basehttp 124 "GET /admin HTTP/1.1" 404 6779
WARNING 2018-12-10 10:00:29,258 base 93 Not Found: /favicon.ico
WARNING 2018-12-10 10:00:29,259 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 6797
INFO 2018-12-10 10:04:24,905 basehttp 124 "GET /admin HTTP/1.1" 301 0
INFO 2018-12-10 10:04:25,273 basehttp 124 "GET /admin/ HTTP/1.1" 200 6587
INFO 2018-12-10 10:04:25,357 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 200 16106
INFO 2018-12-10 10:04:25,599 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
INFO 2018-12-10 10:04:25,601 basehttp 124 "GET /static/admin/css/responsive.css HTTP/1.1" 200 17894
INFO 2018-12-10 10:04:25,669 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
INFO 2018-12-10 10:04:25,704 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2018-12-10 10:04:25,920 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
INFO 2018-12-10 10:04:26,011 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2018-12-10 10:04:26,015 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
INFO 2018-12-10 10:04:26,029 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
INFO 2018-12-10 10:06:18,926 basehttp 124 "GET /admin/goods/brand/ HTTP/1.1" 200 3245
INFO 2018-12-10 10:06:18,960 basehttp 124 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
INFO 2018-12-10 10:06:19,060 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:06:19,256 basehttp 124 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
INFO 2018-12-10 10:06:19,275 basehttp 124 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
INFO 2018-12-10 10:06:19,276 basehttp 124 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6897
INFO 2018-12-10 10:06:19,276 basehttp 124 "GET /static/admin/js/core.js HTTP/1.1" 200 7134
INFO 2018-12-10 10:06:19,296 basehttp 124 "GET /static/admin/js/actions.js HTTP/1.1" 200 6502
INFO 2018-12-10 10:06:19,365 basehttp 124 "GET /static/admin/js/urlify.js HTTP/1.1" 200 8729
INFO 2018-12-10 10:06:19,563 basehttp 124 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
INFO 2018-12-10 10:06:19,608 basehttp 124 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
INFO 2018-12-10 10:06:19,945 basehttp 124 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2018-12-10 10:06:21,558 basehttp 124 "GET /admin/ HTTP/1.1" 200 6587
INFO 2018-12-10 10:06:22,933 basehttp 124 "GET /admin/goods/goodscategory/ HTTP/1.1" 200 3291
INFO 2018-12-10 10:06:23,098 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:06:24,832 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5174
INFO 2018-12-10 10:06:24,860 basehttp 124 "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
INFO 2018-12-10 10:06:24,903 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:06:25,159 basehttp 124 "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
INFO 2018-12-10 10:06:25,159 basehttp 124 "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
INFO 2018-12-10 10:06:25,195 basehttp 124 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
INFO 2018-12-10 10:29:15,584 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:29:15,634 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5457
INFO 2018-12-10 10:29:15,838 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:29:16,218 basehttp 124 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2018-12-10 10:29:31,637 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:29:31,692 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5500
INFO 2018-12-10 10:29:31,866 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:29:52,390 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:29:52,455 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5543
INFO 2018-12-10 10:29:52,614 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:30:07,705 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:30:07,788 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5574
INFO 2018-12-10 10:30:07,946 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:30:21,997 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:30:22,070 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5623
INFO 2018-12-10 10:30:22,187 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:30:30,888 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:30:31,016 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5660
INFO 2018-12-10 10:30:31,069 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:30:50,260 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:30:50,363 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5700
INFO 2018-12-10 10:30:50,446 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:30:56,702 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:30:56,783 basehttp 124 "GET /admin/goods/goodscategory/ HTTP/1.1" 200 6237
INFO 2018-12-10 10:30:56,932 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:32:47,428 basehttp 124 "GET /admin/goods/goodscategory/add/ HTTP/1.1" 200 5497
INFO 2018-12-10 10:32:47,462 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:32:55,395 basehttp 124 "POST /admin/goods/goodscategory/add/ HTTP/1.1" 302 0
INFO 2018-12-10 10:32:55,777 basehttp 124 "GET /admin/goods/goodscategory/ HTTP/1.1" 200 6475
INFO 2018-12-10 10:32:56,134 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7678
INFO 2018-12-10 10:34:03,669 basehttp 124 "GET / HTTP/1.1" 200 5710
WARNING 2018-12-10 10:34:06,340 base 93 Not Found: /categories/9
WARNING 2018-12-10 10:34:06,342 basehttp 124 "GET /categories/9 HTTP/1.1" 404 6906
WARNING 2018-12-10 10:34:18,330 base 93 Not Found: /categories/9/
WARNING 2018-12-10 10:34:18,331 basehttp 124 "GET /categories/9/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:35:02,836 base 93 Not Found: /categories/9/
WARNING 2018-12-10 10:35:02,837 basehttp 124 "GET /categories/9/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:35:10,448 base 93 Not Found: //categories/9/
WARNING 2018-12-10 10:35:10,448 basehttp 124 "GET //categories/9/ HTTP/1.1" 404 6912
WARNING 2018-12-10 10:35:10,578 base 93 Not Found: /favicon.ico
WARNING 2018-12-10 10:35:10,579 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 6903
WARNING 2018-12-10 10:35:14,057 base 93 Not Found: /categories/9/
WARNING 2018-12-10 10:35:14,059 basehttp 124 "GET /categories/9/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:35:26,722 base 93 Not Found: /categories/1/
WARNING 2018-12-10 10:35:26,723 basehttp 124 "GET /categories/1/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:35:29,878 base 93 Not Found: /categories/2/
WARNING 2018-12-10 10:35:29,878 basehttp 124 "GET /categories/2/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:35:32,551 base 93 Not Found: /categories/3/
WARNING 2018-12-10 10:35:32,553 basehttp 124 "GET /categories/3/ HTTP/1.1" 404 6909
WARNING 2018-12-10 10:39:59,062 base 93 Not Found: /categories/3/
WARNING 2018-12-10 10:39:59,064 basehttp 124 "GET /categories/3/ HTTP/1.1" 404 6908
ERROR 2018-12-10 10:40:44,640 exception 118 Internal Server Error: /categories/3/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\meiduo\apps\goods\views.py", line 19, in get
    cat1 = cat2.parent
AttributeError: 'NoneType' object has no attribute 'parent'
ERROR 2018-12-10 10:40:44,988 basehttp 124 "GET /categories/3/ HTTP/1.1" 500 95814
INFO 2018-12-10 10:40:49,934 basehttp 124 "GET /categories/9/ HTTP/1.1" 200 5604
INFO 2018-12-10 10:40:49,982 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2018-12-10 10:40:49,984 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2018-12-10 10:40:49,984 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2018-12-10 10:40:49,985 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,058 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,076 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,079 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,080 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,084 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,173 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2018-12-10 10:40:50,340 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2018-12-10 12:01:20,586 basehttp 124 "GET /categories/1/skus/ HTTP/1.1" 200 5662
INFO 2018-12-10 12:02:10,513 basehttp 124 "GET /categories/1/skus/ HTTP/1.1" 200 5662
INFO 2018-12-10 14:55:43,231 basehttp 124 "GET / HTTP/1.1" 200 5851
ERROR 2018-12-10 14:55:45,195 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 14:55:45,555 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121152
INFO 2018-12-10 15:05:04,213 basehttp 124 "GET / HTTP/1.1" 200 5851
INFO 2018-12-10 15:05:04,244 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,397 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,699 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,700 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,701 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,731 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,732 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
INFO 2018-12-10 15:05:04,733 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2018-12-10 15:05:05,005 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
ERROR 2018-12-10 15:05:06,940 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:05:07,228 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121151
ERROR 2018-12-10 15:06:20,218 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:06:20,532 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121144
ERROR 2018-12-10 15:06:24,875 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:06:24,921 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121271
WARNING 2018-12-10 15:06:25,068 base 93 Not Found: /favicon.ico
WARNING 2018-12-10 15:06:25,069 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 7930
INFO 2018-12-10 15:06:43,143 basehttp 124 "GET / HTTP/1.1" 200 5851
ERROR 2018-12-10 15:06:44,864 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:06:45,178 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121152
ERROR 2018-12-10 15:09:29,115 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:09:29,423 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121144
INFO 2018-12-10 15:09:39,955 basehttp 124 "GET / HTTP/1.1" 200 5851
ERROR 2018-12-10 15:09:43,066 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:09:43,112 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121152
ERROR 2018-12-10 15:13:04,043 exception 118 Internal Server Error: /skus/search/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\rest_framework\mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\drf_haystack\generics.py", line 55, in get_queryset
    queryset = queryset.models(*self.index_models)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\query.py", line 372, in models
    if model not in connections[self.query._using].get_unified_index().get_indexed_models():
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 306, in get_indexed_models
    return list(self.get_indexes().keys())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-10 15:13:04,350 basehttp 124 "GET /skus/search/ HTTP/1.1" 500 121412
WARNING 2018-12-10 15:13:04,510 base 93 Not Found: /favicon.ico
WARNING 2018-12-10 15:13:04,511 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 7930
WARNING 2018-12-11 10:02:19,739 base 93 Not Found: /cart
WARNING 2018-12-11 10:02:19,752 basehttp 124 "GET /cart HTTP/1.1" 404 8389
WARNING 2018-12-11 10:02:19,828 base 93 Not Found: /favicon.ico
WARNING 2018-12-11 10:02:19,829 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 8410
WARNING 2018-12-11 10:02:22,546 base 93 Not Found: /cart/
WARNING 2018-12-11 10:02:22,548 basehttp 124 "GET /cart/ HTTP/1.1" 404 8392
WARNING 2018-12-11 10:03:24,094 base 93 Not Found: /cart/
WARNING 2018-12-11 10:03:24,097 basehttp 124 "GET /cart/ HTTP/1.1" 404 8392
INFO 2018-12-11 10:03:58,195 basehttp 124 "GET /cart/ HTTP/1.1" 200 9581
INFO 2018-12-11 10:03:58,220 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-12-11 10:03:58,518 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-12-11 10:03:58,518 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-12-11 10:03:58,526 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-12-11 10:03:58,538 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-12-11 10:03:58,548 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-12-11 10:03:58,553 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-12-11 10:03:58,829 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-12-11 10:03:58,829 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-12-11 10:03:58,834 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-12-11 10:03:58,851 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2018-12-11 10:04:38,545 basehttp 124 "GET /api-auth/login/?next=/cart/ HTTP/1.1" 200 2707
ERROR 2018-12-11 10:04:45,749 exception 118 Internal Server Error: /api-auth/login/
Traceback (most recent call last):
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\views.py", line 65, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\views\generic\edit.py", line 142, in post
    return self.form_valid(form)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\views.py", line 94, in form_valid
    auth_login(self.request, form.get_user())
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\__init__.py", line 161, in login
    user_logged_in.send(sender=user.__class__, request=request, user=user)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\dispatch\dispatcher.py", line 178, in send
    for receiver in self._live_receivers(sender)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\dispatch\dispatcher.py", line 178, in <listcomp>
    for receiver in self._live_receivers(sender)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\models.py", line 20, in update_last_login
    user.save(update_fields=['last_login'])
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\contrib\auth\base_user.py", line 73, in save
    super().save(*args, **kwargs)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\base.py", line 729, in save
    force_update=force_update, update_fields=update_fields)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\db\models\base.py", line 769, in save_base
    update_fields=update_fields, raw=raw, using=using,
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\dispatch\dispatcher.py", line 178, in send
    for receiver in self._live_receivers(sender)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\django\dispatch\dispatcher.py", line 178, in <listcomp>
    for receiver in self._live_receivers(sender)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\signals.py", line 51, in handle_save
    index = self.connections[using].get_unified_index().get_index(sender)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 316, in get_index
    indexes = self.get_indexes()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 300, in get_indexes
    self.build()
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 238, in build
    self.collect_fields(index)
  File "F:\Pycharm\Project\meiduo_env\lib\site-packages\haystack\utils\loading.py", line 243, in collect_fields
    for fieldname, field_object in index.fields.items():
AttributeError: 'SKUIndex' object has no attribute 'fields'
ERROR 2018-12-11 10:04:46,209 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 500 196370
INFO 2019-01-02 16:35:34,456 basehttp 124 "GET / HTTP/1.1" 200 5623
WARNING 2019-01-02 16:35:39,514 basehttp 124 "GET /address/ HTTP/1.1" 401 5152
INFO 2019-01-02 16:35:41,645 basehttp 124 "GET /api-auth/login/?next=/address/ HTTP/1.1" 200 2710
INFO 2019-01-02 16:35:42,914 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2944
INFO 2019-01-02 16:35:46,377 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2944
INFO 2019-01-02 16:35:51,292 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2944
INFO 2019-01-02 16:35:54,768 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2944
INFO 2019-01-02 16:38:08,773 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2944
INFO 2019-01-02 16:38:16,629 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2945
INFO 2019-01-02 16:38:18,867 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2945
INFO 2019-01-02 16:40:38,565 basehttp 124 "GET / HTTP/1.1" 200 5623
INFO 2019-01-02 16:40:40,561 basehttp 124 "GET /api-auth/login/?next=/ HTTP/1.1" 200 2702
INFO 2019-01-02 16:40:41,853 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2936
INFO 2019-01-02 16:40:45,677 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2936
INFO 2019-01-02 16:40:52,482 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2936
INFO 2019-01-02 16:40:57,622 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2937
INFO 2019-01-02 16:41:03,572 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2937
INFO 2019-01-02 16:41:05,312 basehttp 124 "POST /api-auth/login/ HTTP/1.1" 200 2937
INFO 2019-01-02 16:43:38,747 basehttp 124 "GET /address HTTP/1.1" 301 0
WARNING 2019-01-02 16:43:38,764 basehttp 124 "GET /address/ HTTP/1.1" 401 5152
INFO 2019-01-02 16:43:53,632 basehttp 124 "GET /address/areadump HTTP/1.1" 301 0
WARNING 2019-01-02 16:43:53,646 basehttp 124 "GET /address/areadump/ HTTP/1.1" 401 5351
